.travis.yml

```diff
@@ -9,5 +9,5 @@ script:
   - pytest
 
 install:
-  - pip install pipenv
-  - pipenv sync --dev --verbose
+  - pip install poetry
+  - pipenv install --with dev --verbose --verbose
```

Comment:
```suggestion
  - poetry install --with dev --verbose
```

Reply:
Yeah, I left there `pipenv` instead of `poetry`.... I solved that...

Double `--verbose` is intentional. Would you like to have it only once there?
https://python-poetry.org/docs/cli/#global-options

Reply:
Now I am thinking about it, let's have only one there :-)